src/components/UI/ALibrary/AButton.tsx

```diff
@@ -29,7 +31,8 @@ export default function AButton({
   testID = null,
 }: AButtonType) {
   const { colors } = useThemeColors();
-  const backgroundColor = colors[type];
+  const selectedBrandStyle = useSelector((state: RootState) => state.configuration.selectedBrandStyle);
```

Comment:
same comment here we need to default to `colors.brandStyleOrange`

Reply:
Updated it. 55b5dc9f0af99d1f85951d3704744a830f4324ee a22bf175aec8ba26e175b450add53bc58bd3e3dc